File: example-viewbinding/src/main/java/com/xwray/groupie/example/viewbinding/item/FullBleedCardItem.java
package com.xwray.groupie.example.viewbinding.item;

import androidx.annotation.ColorRes;

import com.xwray.groupie.example.viewbinding.MainActivity;

public class FullBleedCardItem extends CardItem {

    public FullBleedCardItem(@ColorRes int colorRes) {
        super(colorRes);
        getExtras().remove(MainActivity.INSET_TYPE_KEY);
    }
}

File: example-viewbinding/src/main/java/com/xwray/groupie/example/viewbinding/item/FullBleedCardItem.kt
package com.xwray.groupie.example.viewbinding.item

import androidx.annotation.ColorInt
import com.xwray.groupie.example.viewbinding.INSET_TYPE_KEY

class FullBleedCardItem(@ColorInt colorInt: Int) : CardItem(colorInt) {
  init {
    extras.remove(INSET_TYPE_KEY)
  }
}
